### DISCUSSION TITLE
Sortierung auch für den Bereich "Gesichter"

### DISCUSSION THREAD

**Discussion by kleinrene** [bpatrik/pigallery2#315] [Ideas]
Since I only have rudimentary knowledge of software development, here is a first idea for improvement:

The bar in the "Images" area with the sorting function would also be helpful in the "Faces" area.

**Comment by bpatrik** [comment]
Yeah this is a good idea. Unfortunately, I don't think I will have time for this, but feel free to jump in, I can give some pointers if needed
